### ISSUE TITLE
Tabs for Multiple Graph Viewers

### ISSUE THREAD

**Issue by avose** [avose/GLShell#15]
It would be good to allow multiple graph views to be running at the same time, maybe only render one at a time, but have tabs to switch between them.

The physics might only run on the current tab to save CPU cycles.. Maybe make it an option for the user. If they want to run on all tabs, go for it.

**Comment by avose** [comment]
I added a notebook on the left / graph side of the IDE. This holds the OpenGL graph view in a tab, and adds new tabs to display search results. The search result tabs aren't quite finished yet.

One graph in a tab with the search result tabs works just fine. However, when I try to make a second glsGraphCanvas and add it to the notebook, there are issues with rendering being slow / non-responsive. Additionally, when switching tabs between two glsGraphCanvas tabs, things seem to lock up.

I think the issue might be related to the two OpenGL contexts fighting one another. If I bind an event handler to the notebook to catch the page change event, I may be able to disable rendering on the tab that's being put in the background, and then enable rendering on the tab coming into selection and make it's OpenGL context the current one. If I manually catch the events and disable / enable things, I'm hopeful that multiple OpenGL panels will work out OK.

I also had to make the glsGraphPanel delay the creation of the glsGraphCanvas in order for it to work in a notebook page at all. After the glsGraphPanel is added to the notebook, then the glsGraphCanvas can be created. So, there is certainly some friction with putting the OpenGL canvas in a notebook. However, again, I'm hopeful that manually doing some disable / enable work can get this all going in the end.
